src/com/egeniq/widget/CustomTypefaceSpan.java

```diff
@@ -0,0 +1,48 @@
+package com.egeniq.widget;
+
+import android.graphics.Paint;
+import android.graphics.Typeface;
+import android.text.TextPaint;
+import android.text.style.MetricAffectingSpan;
+
+/**
+ * Created by josvanegmond on 7/27/15.
+ * Assisting class for multiple typefaces
+ * Original code from http://stackoverflow.com/a/17961854
+ */
+public class CustomTypefaceSpan extends MetricAffectingSpan {
```

Comment:
This is exactly the source code of TypefaceSpan (http://grepcode.com/file/repository.grepcode.com/java/ext/com.google.android/android/5.1.1_r1/android/text/style/TypefaceSpan.java#TypefaceSpan). Maybe you can remove this class and use the built-in one?